test/extended/networking.sh

```diff
@@ -161,10 +161,10 @@ function test-osdn-plugin() {
 
   if deploy-cluster "${name}" "${plugin}" "${log_dir}"; then
     os::log::info "Running networking e2e tests against the ${name} plugin"
-    export TEST_REPORT_FILE_NAME="${name}-junit"
 
     local kubeconfig="$(get-kubeconfig-from-root "${OPENSHIFT_CONFIG_ROOT}")"
-    if ! TEST_REPORT_FILE_NAME=networking_${name}_${isolation} \
+    if ! TEST_REPORT_FILE_NAME="networking_${name}_${isolation}-junit" \
```

Comment:
I don't know what any of this does but FWIW I don't get why we include ${isolation} in the file name since ${name} is already unique (and more obviously implies whether isolation is in effect than having a random "true" or "false" in the filename does).